app/helpers/layout_helper.rb

```diff
@@ -0,0 +1,5 @@
+module LayoutHelper
+  def show_nav_menu?
+    logged_in? && !home_pages? || home_page? && logged_in?
```

Comment:
Is it clearer with `logged_in? && (home_page? || !home_pages?)`?

Reply:
👍